GVRf/Framework/backend_daydream/src/main/jni/daydream_renderer.cpp

```diff
@@ -104,9 +81,9 @@ DaydreamRenderer::~DaydreamRenderer() {
 void DaydreamRenderer::InitializeGl() {
     gvr_api_->InitializeGl();
 
-    // Because we are using 2X MSAA, we can render to half as many pixels and
-    // achieve similar quality.
-    render_size_ = HalfPixelCount(gvr_api_->GetMaximumEffectiveRenderTargetSize());
+    //@todo read gvr.xml and obtain the values from EyeBufferParams
+    render_size_.height = kDefaultFboResolution;
+    render_size_.width = 2*kDefaultFboResolution;
```

Comment:
We render though either to left or to the right half of these. So they have to be sized to accommodate two eyes.